go.mod

```diff
@@ -13,11 +13,12 @@ require (
 	github.com/mjl-/sherpats v0.0.6
 	github.com/prometheus/client_golang v1.18.0
 	github.com/russross/blackfriday/v2 v2.1.0
+	github.com/saintfish/chardet v0.0.0-20230101081208-5e3ef4b5456d
```

Comment:
I don't see this is used? Was it for testing?

Reply:
Can be mistake. I will check.